packages/@lwc/template-compiler/src/shared/types.ts

```diff
@@ -28,6 +29,7 @@ export enum LWCDirectiveRenderMode {
 export interface BaseNode {
     type: string;
     location: SourceLocation;
+    _original?: parse5.CommentNode | parse5.TextNode | parse5.Element;
```

Comment:
IIRC we had a reason we wanted to get rid of `_original`? Is it necessary to bring it back?

Reply:
I ended up doing a custom serializer for our generated ast nodes (https://github.com/salesforce/lwc/pull/2781/commits/2f611367e9fb18e7c8d7e04eb327da4e780e2d23) so this is removed.